gradleUtils/PatchedArtifactsBuild.gradle

```diff
@@ -0,0 +1,34 @@
+def config = project.gradle.settings.extensions.findByName('patchedArtifacts')
```

Comment:
It would be great to include a comment in this file explaining what it's for and how it works in a sentence or two.